examples/universal-cnf/vppagent/cmd/config/composite.go

```diff
@@ -68,19 +56,23 @@ func (uce *UniversalCNFEndpoint) Request(ctx context.Context,
 		return nil, err
 	}
 
-	if err := action.Process(uce.backend, uce.nsmClient); err != nil {
+	if err := action.Process(ctx, uce.backend, uce.nsmClient); err != nil {
 		logrus.Errorf("Failed to process: %+v", uce.endpoint.Action)
 		return nil, err
 	}
-	return conn, nil
+
+	if endpoint.Next(ctx) != nil {
+		return endpoint.Next(ctx).Request(ctx, request)
+	}
+	return request.GetConnection(), nil
```

Comment:
Hmm, should that be `return conn, nil`?

Reply:
It is same, but it would be cleaner to return conn. Thanks.